types/text-table/index.d.ts

```diff
@@ -20,7 +20,7 @@ declare namespace table {
         hsep?: string;
 
         /** An array of alignment types for each column, default ['l','l',...]. */
-        align?: Array<'l' | 'r' | 'c' | '.'>;
+        align?: Array<'l' | 'r' | 'c' | '.' | null | undefined>;
```

Comment:
```suggestion
        align?: Array<'l' | 'r' | 'c' | '.' | null>;
```

I think the `?` in the property name for `align` already allows `undefined`, but adding the `null` looks good to me.

Reply:
no so, because the `?` just says it can either be missing entirely, or it can be an array containing those possible values.  It does not allow one of the values of the array to be `undefined`.